docs/sql-ref-syntax-aux-conf-mgmt-set.md

```diff
@@ -42,7 +42,7 @@ SET property_key[ =property_value ];
 
 <dl>
   <dt><code><em>property_key=property_value</em></code></dt>
-  <dd>Sets the value of specified property key.</dd>
+  <dd>The new value overrides any existing value for the specified property key.</dd>
```

Comment:
@sharangk This makes it sound like it only supports a update op ? How about the following ?
```
Sets the value for a given property key. If an old value exists for a given property key, then it gets overridden by the new value.
```

Reply:
Done. Thanks.